Senla/RPS.swift

```diff
@@ -0,0 +1,93 @@
+import Foundation
+class RPS{
+    enum variants{
+        case Rock
+        case Paper
+        case Scissors
+    }
+    var pChoose=variants.Rock
+    var cChoose:variants=variants.Rock
+    var draw_available=true;
```

Comment:
никаких нижних подчеркиваний в неймингах, только camelCase
```suggestion
    var drawAvailable=true;
```